ipn/ipnlocal/profiles.go

```diff
@@ -452,20 +452,34 @@ var defaultPrefs = func() ipn.PrefsView {
 	prefs.LoggedOut = true
 	prefs.WantRunning = false
 
-	prefs.ControlURL = winutil.GetPolicyString("LoginURL", "")
+	if controlURL, err := winutil.GetPolicyString("LoginURL"); err == nil {
+		prefs.ControlURL = controlURL
+	} else {
+		prefs.ControlURL = ""
+	}
```

Comment:
Equivalent but shorter and matches the style you are using elsewhere

```suggestion
	controlURL, _ := winutil.GetPolicyString("LoginURL")
	prefs.ControlURL = controlURL
```